pipeline-automation-scripts/sql_import/readme.md

```diff
@@ -0,0 +1,17 @@
+# Upload TPCH Data in sql server
+This script used to upload data generated from TPCH tool into sql server using Bulk Copy Program Utility[(BCP)](https://docs.microsoft.com/en-us/sql/tools/bcp-utility?view=sql-server-ver15)
```

Comment:
What is TPCH tool. How can it be used to generate data? 
How can BCP be used?

Reply:
TPCH tool is basically tpch data generator which used to generate data. How to use this data generator is mentioned in TDD and readme.md also.
This script is basically use bcp command to upload data in sql-server. BCP command use in script itself